cli/replicasets/failover_priority.go

```diff
@@ -21,7 +21,7 @@ func SetFailoverPriority(ctx *context.Ctx, args []string) error {
 		return err
 	}
 
-	if ctx.Replicasets.FailoverPriorityNames, err = common.GetInstancesFromArgs(args, ctx.Project.Name); err != nil {
+	if ctx.Replicasets.FailoverPriorityNames, err = common.GetInstancesFromArgs(args); err != nil {
```

Comment:
So we don't get any additional info for `replicasets` command error? I think we should

Reply:
Done, added for replicasets `join`, `expel` and `set-failover-priority`